packages/world/src/WorldFactory.sol

```diff
@@ -17,8 +17,8 @@ contract WorldFactory is IWorldFactory {
   /// @notice Address of the core module to be set in the World instances.
   IModule public immutable coreModule;
 
-  /// @notice Counter to keep track of the number of World instances deployed.
-  uint256 public worldCount;
+  /// @notice Counters to keep track of the number of World instances deployed per address.
+  mapping(address => uint256) public accountCount;
```

Comment:
we can name mapping params now, which helps to self-document
```suggestion
  mapping(address creator => uint256 worldCount) public accountCount;
```
not blocking, but I kinda feel like `worldCounts` might be clearer than `accountCount`?

Reply:
I agree, then it's like "world counts per account"